operations/node_management/NCN_NIC_Replacement.md

```diff
@@ -0,0 +1,101 @@
+# NCN NIC Replacement
+
+This procedure is for readding a non-compute node (NCN) after NIC replacement. This is similar to the [Replace NCN procedure](Add_Remove_Replace_NCNs/Add_Remove_Replace_NCNs.md#replace-ncn-procedure) but many of the steps can be skipped because only the NIC and its MAC addresses changed.
```

Comment:
maybe use 're-adding' instead of 'readding'. I'm not sure if either are words. Spell check for me flags 'readding'

Reply:
It didn't flag it here